.ci/docker-compose-file/docker-compose-rocketmq.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3.9'
+
+services:
+  mqnamesrv:
+    image: apache/rocketmq:4.9.4
+    container_name: rocketmq_namesrv
+    ports:
+      - 9876:9876
+    volumes:
+      - ./rocketmq/logs:/opt/logs
+      - ./rocketmq/store:/opt/store
+    command: ./mqnamesrv 
+    networks:
+      - emqx_bridge
+
+  mqbroker:
+    image: apache/rocketmq:4.9.4 
+    container_name: rocketmq_broker
+    ports:
+      - 10909:10909
+      - 10911:10911
```

Comment:
I suggest we keep the `ports` option commented out, so that when we run the containers locally they don't occupy those ports by default, but we still have the option to easily uncomment those when developing.